doc/releases/quincy.rst

```diff
@@ -5,6 +5,129 @@ Quincy
 Quincy is the 17th stable release of Ceph.  It is named after Squidward
 Quincy Tentacles from Spongebob Squarepants.
 
+v17.2.1 Quincy
+==============
+
+This is the second stable release of Ceph Quincy.
```

Comment:
Do we want to say "second stable release" or "first backport release"

Reply:
"first bugfix release" like we used for 16.2.1 sounds better to me (https://github.com/ceph/ceph/blob/255b0204d27d4747ef336e26c787e3b576f1ca8b/doc/releases/pacific.rst#v1621-pacific)